### PR TITLE
Add BingSDK header to EntitySearch

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

**Comment by stefanvalianu** [comment]
Looks interesting, I'll make sure we include this in our spec generator. Thanks!

File: specification/cognitiveservices/data-plane/EntitySearch/v1.0/EntitySearch.json [hunk 1]
```diff
@@ -930,5 +931,20 @@
         }
       }
     }
+  },
+  "parameters": {
+    "x-bingapis-sdk" :{
+      "name": "X-BingApis-SDK",
+      "in": "header",
+      "description": "Activate Swagger compliancy",
```
Comment (stefanvalianu) [inline specification/cognitiveservices/data-plane/EntitySearch/v1.0/EntitySearch.json]: nit: compliance

**Review by stefanvalianu — APPROVED** [review]

**Review by amarzavery — APPROVED** [review]

**Comment by marstr** [comment]
There is external context on this one, that @amarzavery is looped in on. This makes him a more appropriate assignee than me.

**Comment by lmazuel** [comment]
@stefanvalianu you generate the Swagger file? So should we merge this one, or you will add this to your generator, and then re-do a PR?

**Comment by stefanvalianu** [comment]
@lmazuel Feel free to complete this PR and check in; I'll just make sure that the generator reflects the changes so people don't have to do manual edits

**Pull request merged**